Docs/deep_linking.md

```diff
@@ -0,0 +1,186 @@
+
+### Wellbores View
+
+```/servers/:serverUrl/wells/:wellUid/wellbores```
+
+### Object Groups
+
+```/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups```
+
+### Objects View
+
+```/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/objects```
+
+### Object View
+
+```/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/objects/:objectUid```
+
+### Log Types View
+
+```/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes```
+
+### Log Objects View
+
+```/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/objects```
+
+### Log Object View
+
+```/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/objects/:objectUid```
+
+### Filtering Rows Of Views
+
+Most pages can receive a filter to filter the rows of the table in that page. Those filters are added at the end of the url.
+For example showing only rows where parameter "name" has value "test":
+
+```?filter={"name":"test"}```
+
+Url encoded version which needs to be used:
+
+```?filter=%7B%22name%22%3A%22test%22%7D```
+
+
+### Log Curve Values View
+
+```/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/objects/:objectUid/curvevalues?startIndex=<start index>&endIndex=<end index>&mnemonics=<mnemonics>```
+
+#### Additional params for limiting the data to the given range and mnemonics
+
+If there is a need to limit the data in result, following parameters need to be filled up:
+
+  - startIndex
+  - endIndex
+  - mnemonics
+
+StartIndex and endIndex could be numbers for depth based logs or datetime in ISO format  - "YYYY-MM-DDTHH:mm:ss.sssZ" for time based logs.
+Note that full url should not exceed the maximum url length accepted by most browsers - in Witsml Explorer we support maximum 2000 characters.
+
+Samples:
+
+##### Time sample:
+
+```?startIndex=2024-09-06T00:00:00Z&endIndex=2024-09-06T00:01:00Z&mnemonics=["mnem1","mnem2","mnem3","mnem4"]```
+
+url encoded version which should be used:
+
+```?startIndex=2024-09-06T00%3A00%3A00Z&endIndex=2024-09-06T00%3A01%3A00Z&mnemonics=%5B%22mnem1%22%2C%22mnem2%22%2C%22mnem3%22%2C%22mnem4%22%5D```
+
+In the sample above we combine mnemonics "mmem1", "mmem2", "mmem3", "mmen4" for the selected time interval
+
+##### Depth sample:
+
+```?startIndex=1&endIndex=4&mnemonics=["mnem1","mnem2","mnem3","mnem4"]```
+
+url encoded version which should be used:
+
+```?startIndex=1&endIndex=5&mnemonics=%5B%22mnem1%22%2C%22mnem2%22%2C%22mnem3%22%2C%22mnem4%22%5D```
+
+### Multi Logs Curve Info List View
+
+```/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/multilogs?logs=<log uids>```
+
+
+#### Additional params for limiting the data in result
+
+If there is a need to limit the in result, following parameter needs to be filled up:
```

Comment:
I think this is required for multi-logs. And in the multi-log case it's maybe not as much to limit the results (as not selecting any value for logs does not implicitly use all logs like it does for the normal log curve values view).

Reply:
done